python/Ganga/GPIDev/Lib/Registry/RegistrySlice.py

```diff
@@ -304,7 +301,8 @@ def copy(self, keep_going):
         return this_slice
 
     def __contains__(self, j):
-        return j.id in self.objects.keys()
+        allObjK = self.objects.keys()
+        return j.id in allObjK
```

Comment:
Why was this split onto two lines? Likewise with the two similar changes below. I feel it makes the code slightly more verbose for no gain.